Formula/name-that-hash.rb

```diff
@@ -0,0 +1,58 @@
+class NameThatHash < Formula
+  include Language::Python::Virtualenv
+
+  desc "Don't know what type of hash it is? Name That Hash will name that hash type!"
```

Comment:
I would personally recommend something more succinct and less conversational, such as "Modern hash identification system" (taken from the README at https://github.com/HashPals/Name-That-Hash).

Reply:
Thanks @alebcay. I've made your suggested changes.